archinstall/lib/locale/locale.py

```diff
@@ -1,61 +0,0 @@
-from typing import Iterator, List
-
-from ..exceptions import ServiceException, SysCallError
-from ..general import SysCommand
-from ..output import error
-
-
-def list_keyboard_languages() -> Iterator[str]:
-	for line in SysCommand("localectl --no-pager list-keymaps", environment_vars={'SYSTEMD_COLORS': '0'}):
-		yield line.decode('UTF-8').strip()
-
-
-def list_locales() -> List[str]:
-	locales = []
-
-	with open('/usr/share/i18n/SUPPORTED') as file:
-		for line in file:
-			if line != 'C.UTF-8 UTF-8\n':
-				locales.append(line.rstrip())
-
-	return locales
```

Comment:
This needs to be moved to `archinstall/lib/locale/utils.py`. This was merged recently to fix #2041, do not delete it.

Reply:
Ahh my bad thanks for spotting this